USERPREFS.md

```diff
@@ -27,6 +27,14 @@ We have four environments with a respective db and heroku app:
 - staging
 - production
 
+## Console
+
+You can spin up a server that servers a console app for an environment by using the CLI from the `hasura/user-prefs` directory. For example:
+
+```
+hasura console --endpoint https://user-prefs-development.herokuapp.com
```

Comment:
maybee we should recommend using  master

Reply:
good point, I'll change it